### ISSUE TITLE
[1000772-31967] Bad Link Out 

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/KnowledgeBase#62]
The following link out has no data.

```{sql}
SELECT DRUG_HOI_RELATIONSHIP, EVIDENCE_TYPE, MODALITY, STATISTIC_VALUE, EVIDENCE_LINKOUT
FROM DRUG_HOI_EVIDENCE
WHERE DRUG_HOI_RELATIONSHIP = '1000772-31967'
AND EVIDENCE_TYPE = 'MEDLINE_MeSH_ClinTrial'
```

This receives a LinkOut of:
<http://dbmi-icode-01.dbmi.pitt.edu/l/index.php?id=pm-mesh-28188>

Which resolves to:
http://virtuoso.ohdsi.org:8890/sparql?default-graph-uri=&query=PREFIX+ohdsi%3A%3Chttp%3A%2F%2Fpurl.org%2Fnet%2Fohdsi%23%3E%0D%0APREFIX+oa%3A%3Chttp%3A%2F%2Fwww.w3.org%2Fns%2Foa%23%3E%0D%0APREFIX+meddra%3A%3Chttp%3A%2F%2Fpurl.bioontology.org%2Fontology%2FMEDDRA%2F%3E%0D%0APREFIX+ncbit%3A+%3Chttp%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23%3E%0D%0APREFIX+foaf%3A+%3Chttp%3A%2F%2Fxmlns.com%2Ffoaf%2F0.1%2F%3E%0D%0APREFIX+dailymed%3A%3Chttp%3A%2F%2Fdbmi-icode-01.dbmi.pitt.edu%2FlinkedSPLs%2Fvocab%2Fresource%2F%3E%0D%0A%0D%0ASELECT+%3Fan+%3Fsource%0D%0AFROM+%3Chttp%3A%2F%2Fpurl.org%2Fnet%2Fnlprepository%2Fohdsi-pubmed-mesh-poc%3E%0D%0AWHERE+{%0D%0A+%3Fan+a+ohdsi%3APubMedDrugHOIAnnotation%3B%0D%0A+++oa%3AhasBody+%3Fbody%3B%0D%0A+++oa%3AhasTarget+%3Ftarget.%0D%0A%0D%0A+%3Fbody+ohdsi%3AImedsDrug+ohdsi%3A1000772.%0D%0A+%3Fbody+ohdsi%3AImedsHoi+ohdsi%3A45614396.%0D%0A%0D%0A+%3Ftarget+ohdsi%3AMeshStudyType+%22case+reports+%28publication+type%29%22.%0D%0A+%3Ftarget+oa%3AhasSource+%3Fsource.%0D%0A}%0D%0A%0D%0A&format=HTML&timeout=0&debug=on

**Comment by rkboyce** [comment]
Corrected in the source but requires reloading to benefit users. Since we have no users of the linkouts, I would like to wait to reload until the SemMed issue is fixed (issue #63). For now, just get the 'qtxt' for the query and replace the study type with the correct one:

'clinical trial (publication type)'

'case reports (publication type)'

'other (publication type)'

**Comment by ericaVoss** [comment]
@anthonysena  just stumbled on this one calling the WebAPI.  Just commenting on the issue as a friendly reminder as we are building out this new version.
